pandas-stubs/core/frame.pyi

```diff
@@ -115,6 +116,7 @@ from pandas.plotting import PlotAccessor
 
 _str = str
 _bool = bool
+_ScalarOrTupleT = TypeVar("ScalarOrTupleT", bound=Scalar | tuple[Scalar, ...])
```

Comment:
Should be `"_ScalarOrTupleT" .  That's causing the failure right now.

Reply:
I updated it to match, but I didn't think the name of a `TypeVar` really mattered aside from debugging?